cmd/edit/machinepool/cmd_test.go

```diff
@@ -0,0 +1,33 @@
+package machinepool
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+)
+
+// Will un-comment and become the cmd test once the work on refactoring this command is finalized
+var _ = Describe("Edit Machinepool", func() {
+
+	/*Context("getNodePoolReplicas", func() {
```

Comment:
Do we want to leave in this commented out test?

Reply:
I would like to- theyre broken with the refactor, would like to do full end-to-end tests with the new runner/testing structure after the next MR which will introduce this new runner